eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/ContentExtension.php

```diff
@@ -208,44 +204,6 @@ public function getTranslatedFieldDefinitionDescription(ValueObject $content, $f
         throw new InvalidArgumentType('$content', 'Content|ContentInfo', $content);
     }
 
-    /**
-     * Gets translated property generic helper.
-     *
-     * For generic use, expects property in singular form. For instance if 'name' is provided it will first look for
-     * getName( $lang ) method, then property called ->names[$lang], in either case look for correct translation.
-     *
-     * Languages will consist of either forced language or current SiteAccess languages list, in addition for property
-     * lookup helper will look for mainLanguage property and use it if either alwaysAvailable property is true or non-
-     * existing.
-     *
-     * @param \eZ\Publish\API\Repository\Values\ValueObject $object Can be any kid of Value object which directly holds the translated data
-     * @param string $property Property name, example 'name', 'description'
-     * @param string $forcedLanguage Locale we want the content name translation in (e.g. "fre-FR"). Null by default (takes current locale)
-     *
-     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentValue If $property does not exists as plural or as method
-     *
-     * @return string|null
-     */
-    public function getTranslatedProperty(ValueObject $object, $property, $forcedLanguage = null)
```

Comment:
Why did you drop it? Seems out of scope.

Reply:
because I dropped  `ez_trans_prop`

Reply:
> because I dropped `ez_trans_prop`

Not following, scope was to unify naming, not drop features. Is there a replacement for this one?